integration-templates/xero/mappers/to-contact.ts

```diff
@@ -0,0 +1,91 @@
+import type { Contact } from '../../models';
+
+export function toContact(xeroContact: any): Contact {
+    // Find Street address & default phone object, if they exist
+    let streetAddress = xeroContact.Addresses.filter((x: any) => x.AddressType === 'POBOX')[0];
+    let defaultPhone = xeroContact.Phones.filter((x: any) => x.PhoneType === 'DEFAULT')[0];
+
+    streetAddress = streetAddress ? streetAddress : {};
+    defaultPhone = defaultPhone ? defaultPhone : {};
+
+    let formattedPhoneNumber: any = null;
+    if (defaultPhone.PhoneNumber) {
+        formattedPhoneNumber = defaultPhone.PhoneCountryCode
+            ? `+${defaultPhone.PhoneCountryCode}${defaultPhone.PhoneAreaCode}${defaultPhone.PhoneNumber}`
+            : `${defaultPhone.PhoneAreaCode}${defaultPhone.PhoneNumber}`;
+    }
+
+    return {
+        id: xeroContact.ContactID,
+        name: xeroContact.Name,
+        external_id: xeroContact.ContactNumber,
+        tax_number: xeroContact.TaxNumber,
+        email: xeroContact.EmailAddress,
+        address_line_1: streetAddress.AddressLine1,
+        address_line_2: streetAddress.AddressLine2,
+        city: streetAddress.City,
+        zip: streetAddress.PostalCode,
+        country: streetAddress.Country,
+        state: streetAddress.Region,
+        phone: formattedPhoneNumber
+    } as Contact;
+}
+
+export function toXeroContact(contact: Contact) {
+    const xeroContact: any = {};
```

Comment:
this breaks the expectation, none of the fields are optional but the final object could be partial